tests/unit/components/about.spec.js

```diff
@@ -0,0 +1,52 @@
+import Vue from 'vue'
+import { mount, createLocalVue } from '@vue/test-utils'
+import Vuetify from 'vuetify'
+import VueX from 'vuex'
+import About from '@/views/About.vue'
+import { pnpStoreModule } from '../../../src/store/pnp'
+
+const localEdgeVersion = require('@/../package.json').version
```

Comment:
@vickywane there is no edge device version shown on the About page. Only the PWA version. Please correct variable names and logic.

Reply:
I have updated the variable name and also stopped setting the state manually.